src/renderer/views/AddProject/subcomponents/PackageManagerPicker/PackageManagerPicker.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box, FormControlLabel, RadioGroup, Typography } from '@mui/material';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  NewPublication,
+  newPublication,
+  setPublicationField,
+} from '../../../../../shared/redux/slices/addPublicationSlice';
+import RadioBtn from '../../../../components/RadioButton/RadioBtn';
+
+const PackageManagerPicker = () => {
+  const { t } = useTranslation();
+  const { packageManager } = useSelector(newPublication);
+  const dispatch = useDispatch();
+
+  const setPackageManager = (value: NewPublication['packageManager']) => {
+    dispatch(setPublicationField({ field: 'packageManager', value }));
+  };
+
+  return (
+    <Box>
+      <Typography variant='subtitle1' component='p' mb={3}>
+        {t('AddProject.PackageManager.message')}
+      </Typography>
+
+      <Box mb={2}>
+        <RadioGroup
+          defaultValue={packageManager}
+          onChange={(e) =>
+            setPackageManager(
+              e.target.value as NewPublication['packageManager']
+            )
+          }
+        >
```

Comment:
@LostInStatic I think it might just be cleaner to have a handler function instead of defining the `onChange` inline. Even though for now we don't even allow a choice here. 😅

```suggestion
  const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
    const value = event.target.value as NewPublication['packageManager'];
    dispatch(setPublicationField({ field: 'packageManager', value }));
  };

  return (
    <Box>
      <Typography variant='subtitle1' component='p' mb={3}>
        {t('AddProject.PackageManager.message')}
      </Typography>

      <Box mb={2}>
        <RadioGroup defaultValue={packageManager} onChange={handleChange}>
```